contracts-periphery/src/test/UmbraBatchSend.t.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "ds-test/test.sol";
+import "forge-std/Vm.sol";
+import "forge-std/stdlib.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../UmbraBatchSend.sol";
+
+contract UmbraBatchSendTest is DSTest, stdCheats {
```

Comment:
Let's create a `DSTestPlus.sol is DSTest, stdCheats` contract, then have all our test files inherit from that. It can live in `src/test/utils`. This way we don't need to duplicate forge-std/Vm initialization in future test contracts

Reply:
Make sure to move the `Vm vm = Vm(HEVM_ADDRESS);` definition out of `UmbraBatchSendTest` and into `DSTestPlus` 🙂